src/components/Button/Button.tsx

```diff
@@ -234,6 +307,7 @@ const Button = forwardRef(
       return (
         <button {...buttonProps}>
           <span className="monday-style-button__loader">
+            {/** @ts-ignore */}
```

Comment:
Why? If you can, it's preferable to add a reason next to each `@ts-ignore` so that we know why it's there.

Reply:
it's your note so I am not sure. that was the explanation you wrote next to it:
<img width="733" alt="צילום מסך 2022-08-23 ב-8 18 14" src="https://user-images.githubusercontent.com/72390374/186075910-50964903-4385-48fd-b194-18cd80b13807.png">

As I understand, it's because we did not convert these components to TS. Am I correct? (If so I will add a note)

Reply:
Added a comment about this